lib/testLauncher/composeConfig.js

```diff
@@ -164,23 +161,25 @@ function deepDirectorySearch(pathToSearch, filename, shouldSearchDeep) {
  */
 function readRcConfig(pathToConfig) {
 	let mainConfigFilePath = '';
-	const foundFiles = [];
+	let shouldContinueSearch = true;
 
 	if (pathToConfig) {
-		foundFiles.push(pathToConfig);
 		mainConfigFilePath = pathToConfig;
 	} else {
 		DEFAULT_PATHS
 			.filter(defaultConfig => defaultConfig.isWindows === IS_WINDOWS)
 			.forEach((defultConfig) => {
```

Comment:
Replace `forEach` with `for of` loop and use `break` keyword for stop iterations instead of using `shouldContinueSearch`